internal/controller/release_controller.go

```diff
@@ -252,7 +252,7 @@ func (r *ReleaseReconciler) reconcileHMCTemplates(ctx context.Context, releaseNa
 		},
 	}
 
-	if releaseName == "" {
+	if releaseName == "" || r.CreateRelease {
```

Comment:
we only need Release to be created as part of the hcm-templates chart during the initial hmc installation. We should probably rename it to avoid confusion (along with `create-management`, `create-template-management`) (although it's mentioned in the variable description)